src/api_callable.ts

```diff
@@ -133,23 +145,29 @@ export class PromiseCanceller<T = any> extends Canceller {
    * @private
    */
   // tslint:disable-next-line variable-name
-  constructor(PromiseCtor) {
+  constructor(PromiseCtor: PromiseConstructor) {
     super();
     this.promise = new PromiseCtor((resolve, reject) => {
-      this.callback = (err, ...args) => {
-        if (err) {
-          reject(err);
-        } else {
-          resolve(args);
-        }
-      };
-    });
+                     this.callback = (err, ...args) => {
+                       if (err) {
+                         reject(err);
+                       } else {
+                         resolve(args);
+                       }
+                     };
+                   }) as CancellablePromise;
     this.promise.cancel = () => {
       this.cancel();
     };
   }
 }
 
+export interface OtherArgs {
```

Comment:
`OtherArgs` seems too general. Can we call this `ApiCallOtherArgs` or something like that?